test/helpers/position_assertions.cc

```diff
@@ -818,7 +818,7 @@ bool containsLine(string_view text, string_view line) {
     }
     const bool startsOnNewLine = pos == 0 || text.at(pos - 1) == '\n';
     const bool endsLine = pos + line.size() == text.size() || text.at(pos + line.size()) == '\n';
-    return startsOnNewLine && endsLine;
+    return (startsOnNewLine && endsLine) || containsLine(text.substr(pos + line.size()), line);
```

Comment:
Why did you need to change this method? It should be left as-is.

Reply:
Ah, this was something I noticed in the hover tests. 
fc4478a8ee312dd93a9cf62d9c880c1b8854e177

If a match is found, it must match the line exactly.
Searching for `bazel` in 
```
bazelgeuse
baz
bazel
```
Would fail, although `bazel` is a valid line -- because the `pos` of `bazel` is within `bazelgeuse`.

I do think using recursion is a bit too much though. Maybe we could store all the `pos`s in a vector and use the same logic?

Reply:
🤦‍♂ oh no, you're right....great catch!

Your solution works fine. Another approach is a loop; something like:

```c++
int pos = 0;
for(int pos = text.find(line, pos); pos != string::npos; pos = text.find(line, pos)) {
  const bool startsOnNewLine = pos == 0 || text.at(pos - 1) == '\n';
  const bool endsLine = pos + line.size() == text.size() || text.at(pos + line.size()) == '\n';
  if (startsOnNewLine && endsLine) {
    return true;
  }
  // Skip over newline before looping
  pos++;
}
return false;
```

Reply:
I prefer the loop actually, will probably have to do use `text.find(line.substr(pos+1))` or something

Reply:
d71f1aa54